src/visit/Visit.java

```diff
@@ -1,4 +1,82 @@
 package src.visit;
 
+import src.users.Doctor;
+import src.users.Patient;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Date;
+import java.util.Objects;
+
 public class Visit {
+
+    private LocalDate date;
+    private LocalTime time;
+    private int duration;
+    private Doctor doctor;
+    private Patient patient;
```

Comment:
add `private int Rating`
accepts int values from 1 to 5 (less than1 -> 1, more than 5->5 , add this logic to setter)